ckan/templates/_extra_translations.html

```diff
@@ -0,0 +1,148 @@
+
+  {{ _('No organization') }}
+  {{ _('No group') }}
+  {{ _('No category') }}
+  {{ _('No theme') }}
+  {{ _('No topic') }}
+  {{ _('No project') }}
+
+  {{ _('Organization not found') }}
+  {{ _('Group not found') }}
+  {{ _('Category not found') }}
+  {{ _('Theme not found') }}
+  {{ _('Topic not found') }}
+  {{ _('Project not found') }}
+
+  {{ _('You are not a member of any organizations') }}
+  {{ _('You are not a member of any groups') }}
+  {{ _('You are not a member of any categorys') }}
+  {{ _('You are not a member of any themes') }}
+  {{ _('You are not a member of any topics') }}
+  {{ _('You are not a member of any projects') }}
+
+  {{ _('There is no description for this organization') }}
+  {{ _('There is no description for this group') }}
+  {{ _('There is no description for this category') }}
+  {{ _('There is no description for this theme') }}
+  {{ _('There is no description for this topic') }}
+  {{ _('There is no description for this project') }}
+
+  {{ _('A little information about my Organization') }}
+  {{ _('A little information about my Group') }}
+  {{ _('A little information about my Category') }}
+  {{ _('A little information about my Theme') }}
+  {{ _('A little information about my Topic') }}
+  {{ _('A little information about my Project') }}
+
+  {{ _('Are you sure you want to delete this Organization') }}
+  {{ _('Are you sure you want to delete this Group') }}
+  {{ _('Are you sure you want to delete this Category') }}
+  {{ _('Are you sure you want to delete this Theme') }}
+  {{ _('Are you sure you want to delete this Topic') }}
+  {{ _('Are you sure you want to delete this Project') }}
+
+  {{ _('There are no organizations associated with this dataset') }}
+  {{ _('There are no groups associated with this dataset') }}
+  {{ _('There are no categorys associated with this dataset') }}
+  {{ _('There are no themes associated with this dataset') }}
+  {{ _('There are no topics associated with this dataset') }}
+  {{ _('There are no projects associated with this dataset') }}
+
+  {{ _('Are you sure you want to delete this Organization? Note*: Deleting cannot be performed while public or private datasets belong to this organization.') }}
+  {{ _('Are you sure you want to delete this Group? Note*: Deleting cannot be performed while public or private datasets belong to this group.') }}
+  {{ _('Are you sure you want to delete this Category? Note*: Deleting cannot be performed while public or private datasets belong to this category.') }}
+  {{ _('Are you sure you want to delete this Theme? Note*: Deleting cannot be performed while public or private datasets belong to this theme.') }}
+  {{ _('Are you sure you want to delete this Topic? Note*: Deleting cannot be performed while public or private datasets belong to this topic.') }}
+  {{ _('Are you sure you want to delete this Project? Note*: Deleting cannot be performed while public or private datasets belong to this project.') }}
+</pre>
```

Comment:
An artificial translation file like this will go out of sync with the actual messages really easily